carbon-design-system/carbon
# refactor(react): move all next stories to react package

This PR updates our project conventions to use `next` folders in the `react` package instead of stories in the `carbon-react` folder. It also moves all new components into `react`, as well, so that they may rely on internals freely instead of having to cross package boundaries.

This change also removes the `start:v11` task as it is no longer needed 👍 

#### Changelog

**New**

**Changed**

- Move component stories from `carbon-react` to `react` and update any imports
- Move Layer component to react
- Move Theme component, useTheme hook to react
- Update re-exports to match new locations

**Removed**

- Remove extra index.js files from `carbon-react`
- Remove `start:v11` task

#### Testing / Reviewing

- Verify moved stories show up in `carbon-react`
- Verify moved stories do not show up in `react` storybook
- Verify packages export components and hooks as-expected
